docs/tutorials/seqcopy.md

```diff
@@ -1,4 +1,5 @@
 # Sequence copy model
+
 This tutorial will show you the basic usage of Sockeye on a on a simple task: copying a sequence.
 We will generate sequences consisting of digits of variable lengths.
 The task is then to train a model that copies the sequence from the source to the target.
```

Comment:
There are two links below that I believe need to be adjusted to the website. Namely:
* `For this tutorial we assume that you have successfully [installed](../../README.md#installation) Sockeye.`
* `his means that even while the model is still training you can use the model folder for translation, as described in the [next section](#3-translation).`